h2o-r/tests/testdir_misc/runit_explain.R

```diff
@@ -107,10 +107,10 @@ explanation_test_automl_regression <- function() {
 
   # Leaderboard slice works
   # test explanation
-  expect_true("H2OExplanation" %in% class(h2o.explain(aml@leaderboard[1:4,], train)))
+  expect_true("H2OExplanation" %in% class(h2o.explain(aml@leaderboard[-1,], train)))
```

Comment:
it's just to prove that you can use slices, right? No other specific reason to remove the first line in those tests.

Reply:
Exactly.